geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionAdvisor.java

```diff
@@ -1179,12 +1180,12 @@ protected void profileRemoved(Profile profile) {
   /**
    * All advise methods go through this method
```

Comment:
Since you're cleaning up code here, this comment does not follow Javadoc syntax requirements (no `@return` or `@param` tags) so either those should be added, or the comment changed to just be a regular comment rather than a javadoc.